### ISSUE TITLE
Error on selecting Work Orders in manufacturing settings

### ISSUE THREAD

**Issue by bhanukiran6864** [odoo/odoo#75718]
Impacted versions: 14.0

Steps to reproduce: Go to Manufacturing settings->Click on Work Orders. 

Current behavior: We are seeing an error popup on clicking work order. Therefore  I am unable to enable work order feature in Manufacturing.

Expected behavior: I should be able to enable work order feature

Video/Screenshot link (optional):
![Screenshot from 2021-08-30 12-13-44](https://user-images.githubusercontent.com/68856953/131297425-5e1d40cf-9703-48d3-8505-4894238b76f9.png)

**Comment by pedrobaeza** [comment]
Hi, thanks for reporting. Are you able to reproduce the problem in runbot infrastructure (http://runbot.odoo.com/runbot - click on the blue door icon of the version row - admin/admin)?

If so, please explain with all the possible detailed steps to get to the error. Not having them, it's very difficult to hunt down the problem. Please don't be shy of adding details, and even better if you provide some screenshots or videos.

If not, this can be due to a configuration change, a custom/community module you have deployed in your DB or an outdated code. Please report back if it's the case.

For keeping a clean house in this issue tracker, if you don't answer in several days, this will be closed assuming it was one of the stated things.

**Comment by bhanukiran6864** [comment]
@pedrobaeza I am able to reproduce this issue on runbot infrastructure. I tested multiple times in runbot infrastructure and only then I created this issue. Infact screenshot attached is from runbot server itself. Since testing is done on runbot server, there is no chance of configuration change/custom module issue.
**Steps to reproduce:**
1. Go to Manufacturing module and click on configuration
2. Select settings
3. Try to enable work orders feature
On trying to enable work orders feature, I am getting below mentioned error:
ValueError: Invalid field 'is_attachment_features' on model 'res.config.settings'
![Screenshot from 2021-08-30 12-36-43](https://user-images.githubusercontent.com/68856953/131299554-d8d92054-f5a5-4f5e-87fe-9ce64cfb540b.png)
